contribs/github-bot/requirements/requirement.go

```diff
@@ -0,0 +1,12 @@
+package requirements
+
+import (
+	"github.com/google/go-github/v64/github"
+	"github.com/xlab/treeprint"
+)
+
+type Requirement interface {
```

Comment:
I like this interface, along with `Condition`. It's very clean

Reply:
Cool thanks :)